spec/syntax.md

```diff
@@ -412,14 +412,36 @@ option = name [s] "=" [s] (literal / variable)
 
 #### Reserved
 
-**_Reserved_** sequences start with a reserved character
-and are intended for future standardization.
+**_Reserved_** annotations start with a reserved character
+and are intended for future standardization
+as well as private implementation use.
 A reserved sequence MAY be empty or contain arbitrary text.
 A reserved sequence does not include any trailing whitespace.
```

Comment:
You are correct to use the term _annotation_ above, but you should be consistent in talking about annotations in the section. This should probably say something like:

```suggestion
A _reserved_ _annotation_ MAY be empty or contain arbitrary text.
This allows maximum flexibility in future standardization. Future definitions 
are expected to define additional semantics and constraints on the contents
of these _annotations_.
Note that a _reserved annotation_ does not include trailing whitespace.
```